alerta/auth/google.py

```diff
@@ -47,6 +47,7 @@ def google():
 
     customer = get_customer(id_token.email, groups=[domain])
 
-    token = create_token(id_token.subject, profile['name'], id_token.email, provider='google', customer=customer,
+    username = profile.get('name', id_token.email.split('@')[0])
```

Comment:
This is very minor, but can you call this variable `name` not `username`.

Calling it `username` is a bit confusing because the `login` parameter (which is next in the list) is assigned to `preferred_username` within the `create_token()` method. Thanks.

Reply:
Agreed :) I will fix it shortly